code/drasil-utils/Utils/Drasil/Fold.hs

```diff
@@ -25,11 +25,12 @@ foldle1 f g (x:y:xs) = foldle f g (f x y) xs
 -- | Helper for formatting constraints.
 foldConstraints :: (Quantity c) => c -> [Constraint] -> Sentence
 foldConstraints _ [] = EmptyS
-foldConstraints c e  = E $ foldl1 ($&&) $ map constraintToExpr e
+foldConstraints c e  = E $ multiExpr $ map constraintToExpr e
   where
     constraintToExpr (Range _ ri)         = realInterval c ri
     constraintToExpr (EnumeratedReal _ l) = isin (sy c) (DiscreteD l)
     constraintToExpr (EnumeratedStr _ l)  = isin (sy c) (DiscreteS l)
+      -- TODO: Should these 2 above be moved into DisplayExpr? 
```

Comment:
This relates to your discussion in https://github.com/JacquesCarette/Drasil/commit/b585428f1626a9a6aebd93bb7309869c7ed720d7#diff-1c876a4c1a461cc7dbbf3a2edfccc788d3979c4c1f436cf377a8ce323aababc5

Should we being trying something like this next?

Reply:
Yes. In particular, the target to `isin` should be a `DisplayExpr`.

Reply:
I gave this a go in ff34fe093b8898383232e2822f478a394243aef9. I'm not too sure if I'm happy with the result. If `IsIn` is one of many future operators that are meant to be something that's only for display and we never generate code for, then it may be the case that DisplayExpr should just fully recreate Expr within DisplayExpr and purely work in terms of DisplayExpr for all things related to printing (of course, excluding code generation).

Reply:
There won't be that many more things in `Expr` that will migrate to `DisplayExpr`. This will all need to be revisited once we go typed, so it's not worth doing anything complicated now.